src/views/SearchRedesign/LanguageFilter.tsx

```diff
@@ -42,7 +42,7 @@ export const LanguageFilter: FunctionComponent = () => {
   return (
     <CheckboxFilter
       data-testid={testIds.languagesFilter}
-      hint="Select one or more programming languages to filter by. Results will match at least one of the selected languages. If no languages are selected, results will not be filtered by langauges."
+      hint="Choose one or more languages. Results include constructs for use with at least one of the selected languages."
```

Comment:
```suggestion
      hint="Choose one or more languages. Results include constructs available in at least one of the selected languages."
```

Reply:
Ahh yeah I don't think we can reword here, this is from a technical writer